components/buttons/buttons.css

```diff
@@ -0,0 +1,176 @@
+/* ---default button */
+.btn {
+    border: none;
+    background-color: var(--ICONS-ORANGE);
+    color: var(--PURE-WHITE);
+    padding: 1.5rem 2rem;
+    cursor: pointer;
+    font-weight: bold;
+    transition: all .5s ease-in-out;
+}
+
+/* PRIMARY BUTTON */
+.btn-pri {
+    background-color: var(--495057);
+    color: var(--PURE-WHITE);
+}
+
+.btn-pri:hover {
+    background: var(--6C757D);
+}
+
+/* SECONDARY BUTTON */
+.btn-sec {
+    background-color: var(--DEE2E6);
+    color: var(--495057);
+}
+
+.btn-sec:hover {
+    background-color: var(--DE2EE6);
+}
+
+/* ERROR BUTTON */
+.btn-error {
+    background-color: var(--ERROR-RED);
+}
+
+.btn-error:hover {
+    background: var(--ERROR-RED-DARK);
+}
+
+/* BUTTON SUCCESS */
+.btn-success {
+    background: var(--SUCCESS-GREEN);
+}
+
+.btn-success:hover {
+    background: var(--SUCCESS-GREEN-DARK);
+}
+
+/* BUTTON OUTLINE */
+.btn-outline {
+    color: var(--ICONS-ORANGE);
+    border: solid var(--ICONS-ORANGE) 2px;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
+    padding: 1rem 2rem;
+}
+
+.btn-outline:hover,
+.btn-outline:focus {
+    background-color: var(--ICONS-ORANGE);
+    color: var(--PURE-WHITE);
+}
+
+.btn-outline-pri {
+    color: var(--PURE-BLACK);
+    border: solid var(--PURE-BLACK) 2px;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
+    padding: 1rem 2rem;
+}
+
+.btn-outline-pri:hover,
+.btn-outline-pri:focus {
+    background-color: var(--PURE-BLACK);
+    color: var(--PURE-WHITE);
+}
+
+.btn-outline-success {
+    color: var(--SUCCESS-GREEN);
+    border: solid var(--SUCCESS-GREEN) 2px;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
+    padding: 1rem 2rem;
+}
+
+.btn-outline-success:hover,
+.btn-outline-success:focus {
+    background-color: var(--SUCCESS-GREEN);
+    color: var(--PURE-WHITE);
+}
+
+.btn-outline-error {
+    color: var(--ERROR-RED);
+    border: solid var(--ERROR-RED) 2px;
+    background-color: var(--PURE-WHITE);
+    border-radius: 4px;
```

Comment:
You can create a variable for border-radius and use it in your CSS. This will ensure consistency and if you want to change it in the future then you'll find it easier to do so.

Reply:
Yeah !  I can do that thanks for pointing this